resources/views/dashboard/controllers/profile.blade.php

```diff
@@ -116,7 +116,8 @@
         <div class="col-sm-4">
             <h4>My Information</h4>
             <br>
-            <p><b>CID:</b> {{ Auth::id() }}</p>
+	    <p><b>CID:</b> {{ Auth::id() }}</p>
+	    <p><b>Name privacy:</b> {{ Auth::user()->name_privacy == 1 ? 'Enabled' : 'Disabled' }} <a class="info-tooltip" href="https://www.vatusa.net/my/profile" target="_blank" data-toggle="tooltip" title="Click here to update - link to VATUSA profile"><i class="fas fa-info-circle"></i></a></p>
```

Comment:
Let's capitalize "Privacy"

Also, I don't think the "- link to VATUSA profile is necessary"... we could just leave this as "Click Here to Update" and I think that would be plenty